### ISSUE TITLE
Program install error!  (KqkbFOA)

### ISSUE THREAD

**Issue by ZeusIssueReporter** [Ekultek/Zeus-Scanner#115]
Zeus version:
`1.0.56`

Firefox version:
`(52, 4)`

Geckodriver version:
`geckodriver-v0.17.0-linux64.tar.gzgeckodriver-v0.17.0-linux64.tar.gzgeckodriver-v0.17.0-linux64.tar.gz`

Error info:
```Traceback (most recent call last):
  File "/home/admin/web/fu.rezlive.com/public_html/sqli1/Zeus-Scanner/var/google_search/search.py", line 259, in parse_search_results
    query_url = get_urls(query, url_to_search, verbose=verbose, user_agent=user_agent, proxy=proxy_string)
  File "/home/admin/web/fu.rezlive.com/public_html/sqli1/Zeus-Scanner/var/google_search/search.py", line 94, in get_urls
    ff_display = Display(visible=0, size=(800, 600))
  File "/usr/lib/python2.7/site-packages/pyvirtualdisplay/display.py", line 34, in __init__
    self._obj = self.display_class(
  File "/usr/lib/python2.7/site-packages/pyvirtualdisplay/display.py", line 52, in display_class
    cls.check_installed()
  File "/usr/lib/python2.7/site-packages/pyvirtualdisplay/xvfb.py", line 38, in check_installed
    ubuntu_package=PACKAGE).check_installed()
  File "/usr/lib/python2.7/site-packages/easyprocess/__init__.py", line 180, in check_installed
    raise EasyProcessCheckInstalledError(self)
EasyProcessCheckInstalledError: cmd=['Xvfb', '-help']
OSError=[Errno 2] No such file or directory
Program install error! 
````

Running details:
`Linux-3.10.0-693.2.2.el7.x86_64-x86_64-with-centos-7.4.1708-Core`

Commands used:
`zeus.py -d site:*.ca AND inurl:php?id=`

Log file info:
```2017-10-19 18:40:23,651;zeus-log;INFO;seems this is your first time running the appication, doing setup please wait...
2017-10-19 18:40:28,138;zeus-log;INFO;installing Xvfb, required by pyvirutaldisplay...
2017-10-19 18:40:28,155;zeus-log;INFO;attempting to get firefox browser version...
2017-10-19 18:40:28,202;zeus-log;INFO;done, continuing process...
2017-10-19 18:40:28,203;zeus-log;INFO;log file being saved to '/home/admin/web/fu.rezlive.com/public_html/sqli1/Zeus-Scanner/log/zeus-log-5.log'...
2017-10-19 18:40:28,203;zeus-log;INFO;using default search engine...
2017-10-19 18:40:28,203;zeus-log;INFO;starting dork scan with query 'site:*.ca'...
2017-10-19 18:40:28,203;zeus-log;WARNING;will not parse webcache URLs...
2017-10-19 18:40:28,203;zeus-log;WARNING;only pulling URLs with GET(query) parameters...
2017-10-19 18:40:28,203;zeus-log;INFO;attempting to gather query URL...
2017-10-19 18:40:28,209;zeus-log;ERROR;[7;31;31msearch.pyc failed to gather the URL from search engine, caught exception 'cmd=['Xvfb', '-help']
OSError=[Errno 2] No such file or directory
Program install error!' exception has been logged to current log file...
Traceback (most recent call last):
  File "/home/admin/web/fu.rezlive.com/public_html/sqli1/Zeus-Scanner/var/google_search/search.py", line 259, in parse_search_results
    query_url = get_urls(query, url_to_search, verbose=verbose, user_agent=user_agent, proxy=proxy_string)
  File "/home/admin/web/fu.rezlive.com/public_html/sqli1/Zeus-Scanner/var/google_search/search.py", line 94, in get_urls
    ff_display = Display(visible=0, size=(800, 600))
  File "/usr/lib/python2.7/site-packages/pyvirtualdisplay/display.py", line 34, in __init__
    self._obj = self.display_class(
  File "/usr/lib/python2.7/site-packages/pyvirtualdisplay/display.py", line 52, in display_class
    cls.check_installed()
  File "/usr/lib/python2.7/site-packages/pyvirtualdisplay/xvfb.py", line 38, in check_installed
    ubuntu_package=PACKAGE).check_installed()
  File "/usr/lib/python2.7/site-packages/easyprocess/__init__.py", line 180, in check_installed
    raise EasyProcessCheckInstalledError(self)
EasyProcessCheckInstalledError: cmd=['Xvfb', '-help']
OSError=[Errno 2] No such file or directory
Program install error! 

2017-10-19 18:41:02,487;zeus-log;INFO;[32mZeus got an unexpected error and will automatically create an issue for this error, please wait...[0m
2017-10-19 18:41:02,487;zeus-log;INFO;[32mgetting authorization...[0m
2017-10-19 18:41:02,511;zeus-log;INFO;[32mextracting traceback from log file...[0m
2017-10-19 18:41:02,511;zeus-log;INFO;[32mattempting to get firefox browser version...[0m
```

**Comment by Ekultek** [comment]
Run `sudo apt-get install xvfb`
Then do the following:
 - `cd ..`
 - `sudo rm -rf zeus-scanner`
 - `git clone https://github.com/ekultek/zeus-scanner.git`
 - `cd zeus-scanner`
 - `sudo python zeus.py`

That should take care of the issue. Seems that the initial fix for this issue didn't work, so I'll have to take a look at it when I have a little bit of time.

**Comment by Ekultek** [comment]
There's suppose to be an auto fix set in place for this, guess I put it in the wrong spot, so that should be fixed via b2ff3da
